src/app/public/modules/sectioned-form/sectioned-form.component.ts

```diff
@@ -58,6 +60,9 @@ import {
 })
 export class SkySectionedFormComponent implements OnInit, OnDestroy, AfterViewChecked {
 
+  @Input()
+  public maintainTabContent: boolean = false;
```

Comment:
Would you mind adding some jsDocs comments for this new property?

Reply:
Good call. I also am renaming `maintainTabContent` to `maintainSectionContent` for sectioned forms (but leaving the vertical tabs change alone).

I hadn't caught this on my initial review but we refer to things in sectioned forms as "sections" not "tabs" on everything else so this will keep that consistency.